src/backend/joanie/core/utils/payment_schedule.py

```diff
@@ -150,3 +164,32 @@ def has_installments_to_debit(order):
     return any(
         is_installment_to_debit(installment) for installment in order.payment_schedule
     )
+
+
+def send_mail_reminder_for_installment_debit(order, installment):
+    """
+    Prepare the context variables for the mail reminder when the next installment debit
+    from the payment schedule will happen for the owner of the order.
+    """
+    with override(order.owner.language):
+        product_title = order.product.safe_translation_getter(
+            "title", language_code=order.owner.language
+        )
+        currency = settings.DEFAULT_CURRENCY
+        days_until_debit = settings.JOANIE_INSTALLMENT_REMINDER_PERIOD_DAYS
+        installment_amount = Money(installment["amount"])
+        subject = _(
+            f"{settings.JOANIE_CATALOG_NAME} - {product_title} - "
+            f"An installment of {installment_amount} {currency} will be debited in "
+            f"{days_until_debit} days."
+        )
+        # pylint: disable=protected-access
+        # ruff : noqa : SLF001
+        BasePaymentBackend._send_mail(
```

Comment:
That's a little bit weird. Calling a payment backend (and almost the Base one) to send a mail not related to a payment. This method should be extracted in a utils in the `core` application IMO

Reply:
I will create the new method in the utils in the `core` application. Indeed, we can extract the _send_mail() method from the `BasePaymentBackend`.